universal-login-example/src/relayer/TokenGrantingRelayer.js

```diff
@@ -0,0 +1,33 @@
+import {waitForContractDeploy} from './utils';
+import Identity from 'universal-login-contracts/build/Identity';
+import Token from '../../build/Token';
+import Relayer from 'universal-login-relayer';
+import ethers, {Wallet, utils} from 'ethers';
+
+class TokenGrantingRelayer extends Relayer {
+  constructor(config, provider = '', deployerPrivateKey = '', tokenContractAddress = '') {
```

Comment:
2) You can skip default values, then arguments will be undefined and lazy loading (|| operator) will work as expected.

Reply:
✅